src/Grid.php

```diff
@@ -196,6 +195,10 @@ public function setModel(\atk4\data\Model $model, $columns = null)
             $this->applySort();
         }
 
+        if ($this->quickSearch && is_array($this->quickSearch)) {
```

Comment:
so you can define fields for quicksearch as parameter while creating Grid and then when you setModel, then it will be converted to actual QuickSearch object? Well .... yeah, why not. :)

Reply:
yes, although it won't work with JsSearch in #368.